edb/server/protocol/pg_ext.pyx

```diff
@@ -1543,6 +1566,100 @@ cdef class PgConnection(frontend.FrontendConnection):
         return qu
 
 
+cdef WriteBuffer remap_arguments(
+    data: bytes,
+    params: list[dbstate.SQLParam] | None,
+    fe_settings: dbstate.SQLSettings
+):    
+    # The "external" parameters (that are visible to the user)
```

Comment:
Probably it is worth declaring some of the variables with c types, so the code uses unboxed ints